cmd/bundle/init.go

```diff
@@ -60,9 +60,11 @@ See https://docs.databricks.com/en/dev-tools/bundles/templates.html for more inf
 	var configFile string
 	var outputDir string
 	var templateDir string
+	var tag string
 	cmd.Flags().StringVar(&configFile, "config-file", "", "File containing input parameters for template initialization.")
 	cmd.Flags().StringVar(&templateDir, "template-dir", "", "Directory path within a Git repository containing the template.")
 	cmd.Flags().StringVar(&outputDir, "output-dir", "", "Directory to write the initialized template to.")
+	cmd.Flags().StringVar(&tag, "tag", "", "Branch or tag to use for template initialization.")
```

Comment:
Why call it tag? It's called `reference` in `git.Clone`, and the `git` command itself takes a `--branch` parameter.

Reply:
I think we can call the parameter `ref`, but have aliases for `--branch` and `--tag` for readability sake.

Reply:
Done. Both flags exist now. Note, the reason we do not have a generic `--reference` flag is because we do not support cloning of specific commits right now.